app/models/automated_checks/claim_verifiers/census_subjects_taught.rb

```diff
@@ -58,10 +58,14 @@ def eligible?
         match_against = case claim.policy
         when EarlyCareerPayments
           early_career_payments_policy_subjects
+        when LevellingUpPremiumPayments
+          levelling_up_premium_payments_policy_subjects
         when StudentLoans
           student_loans_policy_subjects
         end
 
+        return false unless match_against
```

Comment:
I'd have used an `else` in the case statement above for that

Reply:
I have, but was trying out stuff and forgot to change it back.

Reply:
Done in c9a72dd9d6c09b35c54fd0822083b924a7b1d76c